docs/services/awssqs.md

```diff
@@ -4,10 +4,10 @@
 
 This notification service is capable of sending simple messages to AWS SQS queue. 
 
-* `queue` - name of the queue you are intending to send messages to. Can be overwriten with target destination annotation.
+* `queue` - name of the queue you are intending to send messages to. Can be overridden with target destination annotation.
```

Comment:
overwritten and overridden are often used interchangeably, but they aren't quite. If you have one field that overrides another, then you would generally not say it overwrites.